templates/openstack/tutorials.html

```diff
@@ -0,0 +1,275 @@
+        <h5 class="p-muted-heading u-no-margin--bottom">
+          Openstack
+        </h5>
+      </header>
+      <div class="blog-p-card__content">
+        <h3 class="p-heading--three u-no-margin--top u-no-margin--bottom">8. 
+          <a class="p-link--soft" href="/tutorials/learn-how-openstack-manages-various-virtual-network-resources">
+            Network
+          </a>
+        </h3>
+        <p class="u-sv3">Learn how OpenStack manages various virtual network resources.</p>
+      </div>
+    </div>
+    <div class="col-4 blog-p-card--post">
+      <header class="blog-p-card__header--tutorial">
+        <h5 class="p-muted-heading u-no-margin--bottom">
+          Openstack
+        </h5>
+      </header>
+      <div class="blog-p-card__content">
+        <h3 class="p-heading--three u-no-margin--top u-no-margin--bottom">9. 
+          <a class="p-link--soft" href="/tutorials/launch-and-terminate-cloud-instances">
+            Instances
+          </a>
+        </h3>
+        <p class="u-sv3">Launch and delete cloud instances.</p>
+      </div>
+    </div>
+    <div class="col-4 blog-p-card--post">
+      <header class="blog-p-card__header--tutorial">
+        <h5 class="p-muted-heading u-no-margin--bottom">
+          Openstack
+        </h5>
+      </header>
+      <div class="blog-p-card__content">
+        <h3 class="p-heading--three u-no-margin--top u-no-margin--bottom">10. 
+          <a class="p-link--soft" href="/tutorials/learn-how-openstack-manages-different-types-of-storage">
+            Storage
+          </a>
+        </h3>
+        <p class="u-sv3">Learn how OpenStack manages different types of storage.</p>
+      </div>
+    </div>
+    <div class="col-4 blog-p-card--post">
+      <header class="blog-p-card__header--tutorial">
+        <h5 class="p-muted-heading u-no-margin--bottom">
+          Openstack
+        </h5>
+      </header>
+      <div class="blog-p-card__content">
+        <h3 class="p-heading--three u-no-margin--top u-no-margin--bottom">11. 
+          <a class="p-link--soft" href="/tutorials/limit-access-to-resources-by-applying-quotas">
+            Quotas
+          </a>
+        </h3>
+        <p class="u-sv3">Limit access to resources by applying quotas.</p>
+      </div>
+    </div>
+    <div class="col-4 blog-p-card--post">
+      <header class="blog-p-card__header--tutorial">
+        <h5 class="p-muted-heading u-no-margin--bottom">
+          Openstack
+        </h5>
+      </header>
+      <div class="blog-p-card__content">
+        <h3 class="p-heading--three u-no-margin--top u-no-margin--bottom">12. 
+          <a class="p-link--soft" href="/tutorials/tear-down-your-openstack-lab-environment">
+            Teardown
+          </a>
+        </h3>
+        <p class="u-sv3">Tear down your OpenStack lab environment.</p>
+      </div>
+    </div>                                          
+  </div>
+</section>
+
+<section class="p-strip is-deep is-bordered">
+  <div class="u-fixed-width">
```

Comment:
Would this be better? referring to the `Learn more...` section in the `/core/smartstart` page. 
```
 <div class="u-fixed-width">
   <h2 class="u-sv2">Additional resources</h2>
 </div>
```

Reply:
@tytus-kurek what do you think?

Reply:
@petesfrench I'm good with "Learn more"